dali/python/nvidia/dali/backend.py

```diff
@@ -36,8 +36,8 @@ def deprecation_warning(what):
     initialized = True
 
     # py39 warning
-    if sys.version_info[0] == 3 and sys.version_info[1] == 9:
-        deprecation_warning("DALI support for Python 3.9 is experimental and some functionalities "
+    if sys.version_info[0] == 3 and sys.version_info[1] == 10:
+        deprecation_warning("DALI support for Python 3.10 is experimental and some functionalities "
```

Comment:
Just a food for thought. Does the fact that we add a `3.10` support as experimental mean, that `3.9` in no longer experimental? Or maybe `3.9` wasn't experimental, it just had a warning message of being experimental?

Reply:
It was. This PR together with CI changes enables full 3.9 tests.